content/post/data-warehouse-sla-p2.md

```diff
@@ -27,47 +28,61 @@ Here are four examples of services provided with increasing specificity:
 
 
 ### Response time
-When stakeholders report issues, they deserve a response. At minimum you should acknowledge the issues and provide a rough idea of the response time. It is helpful to clarify a rough idea of the response times in your SLA. While you can develop 
+When stakeholders report issues, they deserve a response. At minimum you should acknowledge the issue and provide a rough idea of the response time. I know it may be tempting not to respond to stakeholders when they report an issue, especially when you are underwater. However, a little acknowledgment goes a long way, even if your response is "I hear you. Everything is broken right now. I will get to it as soon as possible".
 
+I recommend defining an objective severity scale with associated response times in your SLA - this will save you from guessing every time you have a data warehouse issue. Typically severity is measured by number of people impacted (employees or customers), lost revenue or importance of the use case (e.g, we need this report for tomorrow's board meeting).
 
-As with any product, there are 
+It may not be a perfect fit, but your Tech team probably has a issue severity rating scale that you can borrow as a starting point. The added bonus of using your Tech team's scale (or a tweaked version of it) is that your company will already be familiar with it and you will need to do less stakeholder training.
 
+In addition to the scope of the problem, it is also helpful to categorize data issues in one of two buckets: stale data or wrong data. An ETL issue may result in stale data in a report sent to everyone in the company. From the perspective of number of people impacted, this would be pretty high severity. However, if everyone is aware that they are seeing yesterday's data there should be less of an issue. The real issue is when your stakeholders use incorrect data and don't know it.  
 
-The most simple problem - stale data, always makes me think of this quote from the late comedian Mitch Hedberg:
+Whenever I encounter a stale data issue, I, always think of this quote from the late comedian Mitch Hedberg:
 
 >"Escalators don't break, they just temporarily become stairs. There should never be a sign that says 'Escalator Broken, Sorry for the Inconvenience'. It should say 'Escalator Temporarily Stairs, Sorry for the Convenience'"
 
-The data warehouse version of the sign is "Data warehouse temporarily has data from January 1, 2015 through two days ago, sorry for the convenience"
-
+The data warehouse version of the sign is "Data warehouse temporarily has data from the launch of our business through two days ago, sorry for the convenience"
 
 
 ### Monitoring process
-How do you implement & enforce your SLA?
+[WIP]
```

Comment:
To remove?

Reply:
Its the last section I need to tackle today